backend/src/gpml/db/browse.sql

```diff
@@ -6,7 +6,7 @@ SELECT DISTINCT ON (t.topic, (COALESCE(t.json->>'start_date', t.json->>'created'
  WHERE 1=1
 --~ (when (seq (:search-text params)) "AND t.search_text @@ to_tsquery(:search-text)")
 --~ (when (seq (:geo-coverage params)) "AND t.geo_coverage IN (:v*:geo-coverage)")
---~ (when (seq (:topic params)) "AND t.topic IN (:v*:topic)")
+--~ (when (not (nil? (:topic params))) "AND t.topic = ANY(ARRAY[:v*:topic]::varchar[])")
```

Comment:
Just as a comment, `(if (seq ,,)` is an *idiom* to know if something is nil or not. - https://github.com/clojure/clojure/search?l=Clojure&q=%22%28if+%28seq%22

Reply:
`(seq #{})` or `(seq '())` also returns `nil`, though? I want to be able to differentiate between `nil` and an `empty` sequence.